backend/tests/apps/mentorship/model/test_mentee_module.py

```
@@ -0,0 +1,71 @@
+
+    @patch("apps.mentorship.models.MenteeModule.objects.create")
+    def test_unique_together_constraint(self, mock_create):
+        
+        mock_create.side_effect = [
+            MagicMock(spec=MenteeModule),
+            MockIntegrityError("Mock Integrity Error"),
+        ]
+        MenteeModule.objects.create(
+            mentee=self.mentee,
+            module=self.module,
+            started_at=datetime(2023, 1, 1, 10, 0, 0),
```

Ruff (0.14.10)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)